app/controllers/template_invocations_controller.rb

```diff
@@ -17,4 +22,56 @@ def show
     @line_sets = @line_sets.drop_while { |o| o['timestamp'].to_f <= @since } if @since
     @line_counter = params[:line_counter].to_i
   end
+
+  def show_template_invocation_by_host
```

Comment:
This endpoint needs to have a permission mapped to it in the security block in engine.rb

Reply:
added and rebased, thanks!